docs/conf.py

```diff
@@ -82,7 +82,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = ['guides/deployment/heroku.rst']
```

Comment:
The correct way is to mark the rest file as `:orphan:`

<img width="823" alt="CleanShot 2022-12-01 at 10 51 54@2x" src="https://user-images.githubusercontent.com/239003/205135981-f9e0b9a7-809d-47ce-8249-064f992350ca.png">

Reply:
Thanks, @1st1. I'll keep this in mind for the future. Wasn't sure the proper way to do this.

I think we might pull back on this, keeping the guide visible and adding a note at the top of the guide, that way we communicate we do intend to continue to support deployment to Heroku. I'll get this revised.